### DISCUSSION TITLE
Apunte no disponible en la pagina

### DISCUSSION THREAD

**Discussion by mbeorlegui** [dds-jv/foro#353] [Consultas generales] [answered]
Buenas, no estoy pudiendo acceder al apunte de [_Patrones creacionales_](https://docs.google.com/document/d/1wuU7VVvf7B4zhBinlc0RGuJs5lHI5-ZD/edit) que está en la [Clase 3 de la bitácora](https://dds-jv.github.io/bitacoras/2022/jueves-m/clase-03/). Me pide que mande solicitud, probablemente no lo tengan habilitado para que los mails UTN puedan acceder de una.

**Comment by gastonprieto** [comment, marked as answer]
buenas @mbeorlegui 

Ya deberia estar solucionado, quedo mal el link, ya lo actualice.
